steely/plugins/github.py

```diff
@@ -8,13 +8,23 @@
 COMMAND = '.steelygh'
 
 
+PROTOCOL = 'https'
+DOMAIN = 'github.com'
+REPOSITORY = 'sentriz/steely'
+
+API_URL = f'{PROTOCOL}://api.{DOMAIN}/repos/{REPOSITORY}'
+REGULAR_URL = f'{PROTOCOL}://{DOMAIN}/{REPOSITORY}'
+
+
 def handle_prs():
-    pulls = requests.get('https://api.github.com/repos/sentriz/steely/pulls').json()
-    return 'Senan, you have {} fucken pr\'s to merge'.format(len(pulls))
+    pulls = requests.get(f'{API_URL}/pulls').json()
+    message = f'Senan, you have {len(pulls)} fucken pr\'s to merge.'
+    message += '\n{REGULAR_URL}/pulls'
```

Comment:
why not: 

```
f'Senan, you have {len(pulls)} fucken pr\'s to merge.\nREGULAR_URL/pulls'
```

Reply:
It goes over 80 characters. I don't mind breaking that rule though.

Reply:
It's the same craic, Save The 80th

Reply:
> resolved